int/api/cmd/execute_function.go

```diff
@@ -41,6 +41,8 @@ func (e *executeFunction) Handle(params operations.CmdExecuteFunctionParams) mid
 		params.Body.Name,
 		args,
 		uint64(params.Body.Coins),
+		uint64(*params.Body.Expiry),
```

Comment:
nice one! this will partially solve https://github.com/massalabs/station/issues/700